.vib/drupal/cypress/cypress/integration/drupal_spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="cypress" />
+import { random } from '../support/utils';
+
+it('allows creating a new user with a profile picture', () => {
```

Comment:
The test itself is okay, but when we only implement one test-case in Cypress we try to cover the key feature of the asset. This time, I feel that creating a post (just like WordPress or Discourse) is more the core of Drupal than adding a user.

Having said that I would either: 
- Rewrite this to test a a post publishing with an image (recommended)
- Keep this test, but add another one for the aforementioned (usually, if can't test all the features in only one test or the asset has more than one key functionality)

Reply:
I've rewritten the test to publish an article instead.